examples/Makefile

```diff
@@ -0,0 +1,39 @@
+#
+# This Makefile is included by a top-level Makefile in each example directory
+#
+# Invocation from the root of the project:
+#   $ make -C examples/simple
+#
+DOCKER_IMAGE_NAME	?= example-golang-$(EXAMPLE)
+DOCKER_IMAGE_TAG	?= $(subst /,-,$(shell git rev-parse --abbrev-ref HEAD))
+
+all: $(EXAMPLE) docker
+
+$(EXAMPLE): main.go Makefile
+	@echo ">> fetching dependencies"
+	@go get -d
+	@echo ">> building $@"
+	@CGO_ENABLED=0 GOOS=linux go build -a -tags netgo -ldflags '-w'
+
+define DockerfileTemplate
+FROM scratch
```

Comment:
`scratch` works, but generally we tend to prefer to have a shell available, that's why we use busybox based images.

Reply:
On the other hand, this is just a tiny example binary...